### DISCUSSION TITLE
Distribute libraries as WASM

### DISCUSSION THREAD

**Discussion by mcandre** [npm/rfcs#366] [General]
Maybe this has been raised before, but what about distributing our JavaScript libraries as WASM binaries on NPM?

This could unlock rewards for network bandwidth storage space, load times. We could even grandfather non-JavaScript packages like Boost or LAPACK into NPM via WASM compilation. What do you think?

**Comment by ljharb** [comment]
You can already do this; npm isn’t involved in the format of basically anything in your package besides package.json.
